### PR TITLE
Add skipping validation

### PR DESCRIPTION
**Pull request by smirnovanv**
Closes issue #11

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
+import dotenv from 'dotenv';
+
 import { createTempFolder } from './createTempFolder';
 import { fetchFile } from './fetchFile';
 import { isValidLink } from './isValidLink';
 import { showData } from './showData';
 
+dotenv.config();
+
 const downloadFilesIntoTempFolder = async (downloadLinks?: string[]) => {
```
Comment (m0sk1t) [inline src/index.ts:12]: if (!downloadLinks?.length) return;

File: src/index.ts [hunk 2]
```diff
@@ -1,10 +1,14 @@
...
   const links: string[] = [];
```
Comment (m0sk1t) [inline src/index.ts:13]: let links

File: src/index.ts [hunk 3]
```diff
@@ -13,9 +17,11 @@ const downloadFilesIntoTempFolder = async (downloadLinks?: string[]) => {
     links.push(...downloadLinks);
   }
 
-  const validLinks = links.filter((link) => isValidLink(link));
+  if(!process.env.SKIP_CHECK) {
+    links.filter((link) => isValidLink(link));
```
Comment (m0sk1t) [inline src/index.ts:21]: links = links.filter

**Review by m0sk1t — CHANGES_REQUESTED** [review]

File: src/index.ts (hunk 1 above)
Comment (smirnovanv) [reply src/index.ts:12]: done

File: src/index.ts (hunk 2 above)
Comment (smirnovanv) [reply src/index.ts]: done

File: src/index.ts (hunk 3 above)
Comment (smirnovanv) [reply src/index.ts]: done

File: src/index.ts [hunk 4]
```diff
@@ -1,21 +1,29 @@
...
+dotenv.config();
+
 const downloadFilesIntoTempFolder = async (downloadLinks?: string[]) => {
-  const links: string[] = [];
+  let links: string[] = [];
   const tempFolderName = createTempFolder();
 
-  if (downloadLinks) {
-    links.push(...downloadLinks);
+  if (!downloadLinks?.length) {
```
Comment (m0sk1t) [inline src/index.ts:16]: should be placed as the first statement to avoid unnecessary code run before and after

**Review by m0sk1t — CHANGES_REQUESTED** [review]
nit: small changes

File: src/index.ts (hunk 4 above)
Comment (smirnovanv) [reply src/index.ts:13]: done

**Review by m0sk1t — APPROVED** [review]
LGTM)

**Pull request merged**